custom_components/tahoma/water_heater_devices/domestic_hot_water_production.py

```diff
@@ -11,28 +11,51 @@
 
 from ..entity import OverkizEntity
 
-STATE_MANUAL = "manual"
-STATE_AUTO = "auto"
+DHWP_TYPE_MURAL = "io:AtlanticDomesticHotWaterProductionV2_MURAL_IOComponent"
+DHWP_TYPE_CE_FLAT_C2 = "io:AtlanticDomesticHotWaterProductionV2_CE_FLAT_C2_IOComponent"
+DHWP_TYPE_MBL = "modbuslink:AtlanticDomesticHotWaterProductionMBLComponent"
 
 OVERKIZ_TO_OPERATION_MODE = {
     OverkizCommandParam.MANUAL_ECO_ACTIVE: STATE_ECO,
-    OverkizCommandParam.MANUAL_ECO_INACTIVE: STATE_MANUAL,
-    OverkizCommandParam.AUTO: STATE_AUTO,
-    OverkizCommandParam.AUTO_MODE: STATE_AUTO,
+    OverkizCommandParam.MANUAL_ECO_INACTIVE: OverkizCommandParam.MANUAL.value,
+    OverkizCommandParam.AUTO: OverkizCommandParam.AUTO.value,
+    OverkizCommandParam.AUTO_MODE: OverkizCommandParam.AUTO.value,
+    OverkizCommandParam.BOOST: OverkizCommandParam.BOOST.value,
 }
 
 OPERATION_MODE_TO_OVERKIZ = {v: k for k, v in OVERKIZ_TO_OPERATION_MODE.items()}
 
+SUPPORT_BOOST_MODE = 8
```

Comment:
Is this a core feature of did we add this ourselves? This won't be allowed in core, thus I think we should try to work around this.

I dont see where `SUPPORT_BOOST_MODE` is actually used by the way (if it is not a core feature).